App.UserSupport/Repositories/InMemoryUserSupportRepository.cs

```diff
@@ -1,15 +1,16 @@
 ﻿using System.Collections.Generic;
 using App.Configuration;
 using App.UserSupport.Models;
+using System.Linq;
 
 namespace App.UserSupport.Repositories
 {
     public interface IHandlingsRepository
     {
-        IEnumerable<string> GetStringListActiveHandlings();
+        IQueryable<Handling> GetHandlings();
```

Comment:
what is the reason of changing return type to IQueryble ?

Reply:
IQueryable has more functional rather than IEnumerable

Reply:
The single difference of IQuaryble from IEnumerable is that iqueryble supports defer execution. This open possibilities for error, because it depends on the dbcontext and could not be executed, when context is disposed. This is the single difference.